analysis/tests/logic_test.py

```diff
@@ -0,0 +1,96 @@
+import json
```

Comment:
Do we need empty `analysis/tests/__init__.py`?

Reply:
@gromakovsky do we have a task to python tests on CI?

Reply:
No, but I want to create one.

Reply:
https://issues.serokell.io/issue/EDNA-95

Reply:
Empty init file makes python think of a directory as of a package. I think when tests are passed on CI it'll be not relevant, but that's just the standard way for Python.